### DISCUSSION TITLE
What function does pywb container perform?

### DISCUSSION THREAD

**Discussion by marleyjaffe** [ArchiveBox/ArchiveBox#1275] [Q&A] [answered]
Hello, I am setting up archivebox using the docker compose example and curious what function the webrecorder/pywb container is/would perform?

**Comment by pirate** [comment, marked as answer]
Pywb is actually a similar project to archivebox made by a friend of mine. It has a few unique features though that make it useful in conjunction with archivebox, it's able to do "proxy recording" and it's able to import WARC files produced by other programs. 

I just included it as an example of how the two could be used together and to promote pywb a bit because it's awesome software. You can check out their docs for more info on how to use it.

**Reply by marleyjaffe** [reply]
Thank you for that info, that's what I gathered after doing more research. Can you explain a bit more on how these two applications can work together?

**Reply by wsdookadr** [reply]
> and it's able to import WARC files produced by other programs.

@pirate Does that mean #160 is complete?

**Reply by pirate** [reply]
No I mean pywb is able to import WARCs from ArchiveBox, not the other way around. ArchiveBox can  display externally imported WARCs found in `archive/*/warc/*.warc.gz` in the UI but doesn't look inside them / use them for anything.
